src/main/sensors/voltage.c

```diff
@@ -269,29 +270,50 @@ void voltageMeterESCInit(void)
 
 void voltageMeterESCRefresh(void)
 {
+#ifdef USE_DSHOT_TELEMETRY
+    // Just check motor 0 EDT data validity
+    if (useDshotTelemetry &&
+        ((dshotTelemetryState.motorState[0].telemetryTypes & (1 << DSHOT_TELEMETRY_TYPE_VOLTAGE)) != 0) &&
+        (dshotTelemetryState.motorState[0].telemetryData[DSHOT_TELEMETRY_TYPE_VOLTAGE] > 0)) {
+            uint32_t accumulatedVoltage = 0;
+            for (int motor = 0; motor < getMotorCount(); motor++) {
+                accumulatedVoltage += dshotTelemetryState.motorState[motor].telemetryData[DSHOT_TELEMETRY_TYPE_VOLTAGE];
+            }
+            voltageMeterESCState.voltageUnfiltered = 25 * accumulatedVoltage / getMotorCount();
+            voltageMeterESCState.voltageDisplayFiltered = pt1FilterApply(&voltageMeterESCState.displayFilter, voltageMeterESCState.voltageUnfiltered);
+    } else
```

Comment:
This dangling-else style is ugly. One alternative is `do { ... break; } while (false);`, but that is a bit cumbersome for only 2 branches.
Another possibility is to use `bool sensorFound = false;; ... if (!sensorFound) { ..`